leaderboard/llvm_codesize/e_greedy/e_greedy.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""ϵ-greedy policy for LLVM codesize."""
+import logging
+import os
+import random
+import sys
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import NamedTuple
+
+from absl import flags
+
+from compiler_gym.envs import CompilerEnv, LlvmEnv
+
+# Import the ../eval_policy.py helper.
+sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + "/..")
+from eval_policy import eval_policy  # noqa pylint: disable=wrong-import-position
```

Comment:
Is there a better way to include this? Probably people would want to copy this as an example. Should I open an issue?

Reply:
Yes, please feel free to open an issue. We could move `eval_policy()` into the compiler_gym package so that so people could use something like:

```py
from compiler_gym.leaderboard.llvm_codesize import eval_policy
```

Reply:
Filed #158 to follow up on this.